content/authors/deryag/_index.md

```diff
@@ -21,26 +21,22 @@ organizations:
   url: ""
 
 # Short bio (displayed in user profile at end of posts)
-#bio: My research interests include distributed robotics, mobile computing and programmable matter.
+bio: I am a Research Assistant at the University of Washington Applied Physics Lab with a focus in multi-year time-series analysis of seawater characteristics and sonar data. 
 
 #interests:
-#- Artificial Intelligence
-#- Computational Linguistics
-#- Information Retrieval
+- Data Science
+- Time-series Analysis
+- Science Communication 
```

Comment:
@dgumustel if you uncomment the #interest that will bring the format to conform with what yml expects, then the build should work :)

Reply:
That worked, thank you!